avogadro/qtplugins/templatetool/templatetoolwidget.cpp

```diff
@@ -73,6 +73,16 @@ TemplateToolWidget::TemplateToolWidget(QWidget* parent_)
             << "7-pbp"
             << "8-sqa";
 
+  m_groups << "amide"
+           << "carboxylate"
+           << "ester"
+           << "ethylene"
+           << "ethyne"
+           << "nitro"
+           << "phenyl"
+           << "phosphate"
+           << "sulfonate" << tr("Other…");
```

Comment:
I&rsquo;m not confident about the ordering.

https://github.com/OpenChemistry/avogadrolibs/blob/c8c64082ed031e82fb5306306a5b47ed65964ca5/avogadro/qtplugins/templatetool/template.qrc#L34-L42

https://github.com/OpenChemistry/avogadrolibs/blob/c8c64082ed031e82fb5306306a5b47ed65964ca5/avogadro/qtplugins/templatetool/template.qrc#L81-L89

The former is not in dictionary order.

Reply:
The order doesn't matter in the `.qrc` file, so please feel free to re-arrange.

Reply:
Thanks, .qrc is sorted.